.github/workflows/build.yml

```diff
@@ -16,5 +16,19 @@ jobs:
         with:
           go-version: '1.20'
 
+      - name: Install Protoc
```

Comment:
Not sure if you've seen my main comment in the previous review, I'll copy it here so we can also discuss it if needed.

I'd rather separate this action into its own workflow, it serves a different purpose.

Reply:
Hey @lovromazgon! Sure, no problems. :+1: